### DISCUSSION TITLE
Change in umask for Large runners

### DISCUSSION THREAD

**Discussion by jpreynat** [community/community#40876] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi 👋 

We've been using the GitHub-hosted runners for a while now, and I recently tried to switch our jobs to use the new large runners.
However, some of our tests have started failing as soon as I made the switch. Those tests include checks on file permissions.

After investigating a bit, it turns out that the umask value is different between the "original" GitHub-hosted runners `ubuntu-latest`, with a umask of 022, and the new `ubuntu-latest-4-cores`, for which the umask is 002.

This looks like a breaking change that can have a big impact on jobs running on GitHub actions.
On my end, I have tested running the `umask 022` command before the `actions/checkout` but that didn't work. But maybe I'm just missing something?

**Comment by ossanna16** [comment]
Hi there @jpreynat and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
